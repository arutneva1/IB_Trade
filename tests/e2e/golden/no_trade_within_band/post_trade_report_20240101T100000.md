| symbol | side | filled_shares | avg_price | notional | residual_drift_bps |
| --- | --- | --- | --- | --- | --- |
| BBB | BUY | 49.0000 | 50.50 | 2474.50 | 21.57 |
| AAA | BUY | 74.0000 | 100.50 | 7437.00 | 36.93 |
