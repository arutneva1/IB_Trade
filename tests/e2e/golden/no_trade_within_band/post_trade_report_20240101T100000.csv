symbol,side,filled_shares,avg_price,notional,residual_drift_bps
BBB,BUY,49.0,50.5,2474.5,21.57
AAA,BUY,74.0,100.5,7437.0,36.93
